storage/postgres.go

```diff
@@ -16,3 +19,28 @@ type DBConfig struct {
 	Password string
 	DBName   string
 }
+
+const pgStr = "host=%v port=%v user=%v password=%v dbname=%v sslmode=disable"
+const DBDriverName = "postgres"
```

Comment:
Do you really need this constant as expected?

Reply:
Actually no, fixed it